### PR TITLE
sync tokenlist with subgraph

### PR DESCRIPTION
**Pull request by tokdaniel**
Question: Did The solvency list change?

### CODE REVIEW

File: packages/tokenlist/public/superfluid.tier-b.tokenlist.json [hunk 1]
```diff
@@ -321,30 +321,6 @@
       "logoURI": "https://raw.githubusercontent.com/superfluid-finance/assets/master/public/tokens/op/icon.svg",
       "tags": ["underlying"]
     },
-    {
-      "address": "0x4caB5B9930210E2EdC6A905B9C75D615872A1A7E",
```
Comment (kasparkallas) [inline packages/tokenlist/public/superfluid.tier-b.tokenlist.json:325]: I wonder why this one disappeared

**Comment by kasparkallas** [comment]
On Polygon, `0xcaa7349cea390f89641fe306d93591f87595dc1f` needs to become "USDC.ex"
On Arbitrum, `0x1dbc1809486460dcd189b8a15990bca3272ee04e` needs to become "USDC.ex"

File: packages/tokenlist/public/superfluid.tier-b.tokenlist.json (hunk 1 above)
Comment (tokdaniel) [reply packages/tokenlist/public/superfluid.tier-b.tokenlist.json:325]: the only reason i can imagine is that it's not in subgraph, and was added manually.

**Comment by vmichalik** [comment]
there are a couple of other new resolver additions over the last few months, does it make sense to add them here too?

**Review by kasparkallas — APPROVED** [review]

**Pull request merged**